cookbook/integrations/external_services/horovod/requirements.txt

```diff
@@ -0,0 +1,135 @@
+#
+# This file is autogenerated by pip-compile with python 3.8
+# To update, run:
+#
+#    /Library/Developer/CommandLineTools/usr/bin/make requirements.txt
+#
+attrs==21.2.0
+    # via scantree
+certifi==2021.5.30
+    # via requests
+charset-normalizer==2.0.4
+    # via requests
+click==7.1.2
+    # via flytekit
+croniter==1.0.15
+    # via flytekit
+dataclasses-json==0.5.5
+    # via flytekit
+decorator==5.0.9
+    # via retry
+deprecated==1.2.13
+    # via flytekit
+dirhash==0.2.1
+    # via flytekit
+diskcache==5.2.1
+    # via flytekit
+docker-image-py==0.1.12
+    # via flytekit
+docstring-parser==0.10
+    # via flytekit
+flyteidl==0.20.2
+    # via flytekit
+flytekit==0.22.1
+    # via
+    #   flytekitplugins-kfmpi
+    #   flytekitplugins-spark
+git+https://github.com/flyteorg/flytekit.git@ac3c84b170cfc658ad78e87e1b2d0da0cfe2f6f7#subdirectory=plugins/flytekit-kf-mpi
+    # via -r requirements.in
+flytekitplugins-spark==0.22.1
+    # via -r requirements.in
+grpcio==1.40.0
+    # via flytekit
+idna==3.2
+    # via requests
+importlib-metadata==4.8.1
+    # via keyring
+keyring==23.1.0
+    # via flytekit
+marshmallow==3.13.0
+    # via
+    #   dataclasses-json
+    #   marshmallow-enum
+    #   marshmallow-jsonschema
+marshmallow-enum==1.5.1
+    # via dataclasses-json
+marshmallow-jsonschema==0.12.0
+    # via flytekit
+mypy-extensions==0.4.3
+    # via typing-inspect
+natsort==7.1.1
+    # via flytekit
+numpy==1.21.2
+    # via
+    #   pandas
+    #   pyarrow
+pandas==1.3.2
+    # via flytekit
+pathspec==0.9.0
+    # via scantree
+protobuf==3.17.3
+    # via
+    #   flyteidl
+    #   flytekit
+py==1.10.0
+    # via retry
+py4j==0.10.9
+    # via pyspark
+pyarrow==3.0.0
+    # via flytekit
+pyspark==3.1.2
+    # via flytekitplugins-spark
+python-dateutil==2.8.1
+    # via
+    #   croniter
+    #   flytekit
+    #   pandas
+python-json-logger==2.0.2
+    # via flytekit
+pytimeparse==1.1.8
+    # via flytekit
+pytz==2018.4
+    # via
+    #   flytekit
+    #   pandas
+regex==2021.8.28
+    # via docker-image-py
+requests==2.26.0
+    # via
+    #   flytekit
+    #   responses
+responses==0.13.4
+    # via flytekit
+retry==0.9.2
+    # via flytekit
+scantree==0.0.1
+    # via dirhash
+six==1.16.0
+    # via
+    #   flytekit
+    #   grpcio
+    #   protobuf
+    #   python-dateutil
+    #   responses
+    #   scantree
+sortedcontainers==2.4.0
+    # via flytekit
+statsd==3.3.0
+    # via flytekit
+typing-extensions==3.10.0.2
+    # via typing-inspect
+typing-inspect==0.7.1
+    # via dataclasses-json
+urllib3==1.26.6
+    # via
+    #   flytekit
+    #   requests
+    #   responses
+wheel==0.37.0
+    # via flytekit
+wrapt==1.12.1
+    # via
+    #   deprecated
+    #   flytekit
+zipp==3.5.0
+    # via importlib-metadata
```

Comment:
can we not checkin the rossmann.tgz file. Can we not just host it in our static contents? It is 7MB, so will really affect flytesnacks github repo

Reply:
Done.